### DISCUSSION TITLE
Welcome to lemmy-ui Discussions!

### DISCUSSION THREAD

**Discussion by dessalines** [LemmyNet/lemmy-ui#1393] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by alectrocute** [comment]
Thanks so much for enabling this.

**Comment by sunaurus** [comment]
Cool!

I'm a software engineer, admin of https://lemm.ee and aspiring contributor to Lemmy code. I'm excited to get to know some more folks here over the coming weeks!

By the way @dessalines: I noticed that you opened this discussion board specifically on the lemmy-ui repo. Is it meant to be scoped just for UI-related discussions?

**Reply by dessalines** [reply]
I spose just UI related discussions. To be honest I regret opening this. If / when we migrate off github, I'm 99% sure the conversations here won't be exportable, like they are for issues.

**Reply by Zetaphor** [reply]
Is there somewhere more preferable to have forum-like discussions that are specifically for the core and UI repos?

**Reply by krestenlaust** [reply]
@dessalines if we keep using issues for implementation details, and only use discussion for ephemeral conversations and QA, then it shouldn't matter

**Comment by Zetaphor** [comment]
Would be great to get this for the core repo as well, there's some interesting questions/discussions happening around building automod that would be best served in a central forum rather than Matrix chat
